lib/scan.ts

```diff
@@ -26,7 +33,7 @@ export async function scan(options: Options): Promise<PluginResponse> {
     debug('%d files found \n', filePaths.length);
     const allSignatures: (SignatureResult | null)[] = await pMap(
       filePaths,
-      getSignature,
+      getSignatureMapper,
```

Comment:
here instead of make a directly call to pMap and getSignatureMapper, we can have the following line below

```
    const allSignatures: (SignatureResult | null)[] = await getSignaturesByAlgorithm(
      filePaths,
    );
```

that will align with the changes suggested in `signatures.ts` see https://github.com/snyk/snyk-cpp-plugin/pull/38/commits/784a5e12fe941917c1527f309dd26d79401a17fe#r711707690